lisa/runner.py

```diff
@@ -302,10 +306,11 @@ def _start_loop(self) -> None:
                             # current runner may not be done, but it doesn't
                             # have task temporarily. The root runner can start
                             # tasks from next runner.
+                            self._log.debug(
```

Comment:
This debug message may print a lot in the log. It should be avoid.

Reply:
It is a bit long but we might miss some information if we limit the size. What do you suggest?

Reply:
We can create a task class, and contains more information in the class. So it can be logged here together. The advantage is, it reduces lines of log.